src/foundry/common/documents/tile.mjs.d.ts

```diff
@@ -0,0 +1,204 @@
+     * @defaultValue `null`
+     */
+    _id: fields.DocumentIdField;
+
+    /**
+     * An image or video texture which this tile displays.
+     * @defaultValue `null`
+     */
+    texture: TextureData<{}, { categories: ["IMAGE", "VIDEO"]; initial: null; wildcard: false }>;
+
+    /**
+     * The pixel width of the tile
+     * @defaultValue `0`
+     */
+    width: fields.NumberField<{
+      required: true;
+      min: 0;
+      nullable: false;
+      step: 0.1;
+    }>;
+
+    /**
+     * The pixel height of the tile
+     * @defaultValue `0`
+     */
+    height: fields.NumberField<{ required: true; min: 0; nullable: false; step: 0.1 }>;
+
+    /**
+     * The x-coordinate position of the top-left corner of the tile
+     * @defaultValue `0`
+     */
+    x: fields.NumberField<{ required: true; integer: true; nullable: false; initial: 0; label: "XCoord" }>;
+
+    /**
+     * The y-coordinate position of the top-left corner of the tile
+     * @defaultValue `0`
+     */
+    y: fields.NumberField<{ required: true; integer: true; nullable: false; initial: 0; label: "YCoord" }>;
+
+    /**
+     * The z-index ordering of this tile relative to its siblings
+     * @defaultValue `100`
+     */
+    z: fields.NumberField<{ required: true; integer: true; nullable: false; initial: 100 }>;
+
+    /**
+     * The angle of rotation for the tile between 0 and 360
+     * @defaultValue `0`
+     */
+    rotation: fields.AngleField;
+
+    /**
+     * The tile opacity
+     * @defaultValue `1`
+     */
+    alpha: fields.AlphaField;
+
+    /**
+     * Is the tile currently hidden?
+     * @defaultValue `false`
+     */
+    hidden: fields.BooleanField;
+
+    /**
+     * Is the tile currently locked?
+     * @defaultValue `false`
+     */
+    locked: fields.BooleanField;
+
+    /**
+     * Is the tile an overhead tile?
+     * @defaultValue `false`
+     */
+    overhead: fields.BooleanField;
+
+    /**
+     * Is the tile a roof?
+     * @defaultValue `false`
+     */
+    roof: fields.BooleanField;
+
+    /**
+     * The tile's occlusion settings
+     * @defaultValue `{ mode: CONST.TILE_OCCLUSION_MODES.FADE, alpha: 0, radius: null }`
+     */
+    occlusion: fields.SchemaField<{
+      /**
+       * The occlusion mode from CONST.TILE_OCCLUSION_MODES
+       * @defaultValue `1`
+       */
+      mode: fields.NumberField<{
+        choices: CONST.TILE_OCCLUSION_MODES[];
+        initial: typeof CONST.TILE_OCCLUSION_MODES.FADE;
+        validationError: "must be a value in CONST.TILE_OCCLUSION_MODES";
+      }>;
+
+      /**
+       * The occlusion alpha between 0 and 1
+       * @defaultValue `0`
+       */
+      alpha: fields.AlphaField<{ initial: 0 }>;
+
+      /**
+       * An optional radius of occlusion used for RADIAL mode
+       * @defaultValue `null`
+       */
+      radius: fields.NumberField<{ positive: true }>;
+    }>;
+
+    /**
+     * The tile's video settings
+     * @defaultValue `null`
```

Comment:
This should be changed. It is at least `{}`, but probably better to refer to the inner properties.

Reply:
Updated to refer to inner props. As it turns out, it's `{"loop": true, "autoplay": true, "volume": 0 }` by default 👀